src/sdkBase.ts

```diff
@@ -127,6 +137,10 @@ export abstract class SdkBase {
     return this._tokenPromiseHandler.createMaybeDeferredPromise(token ?? null);
   }
 
+  public getInitCallbacks() {
```

Comment:
I don't think this should be a public method. If this is mainly for testing, I suggested an alternative way to test it above.